pre-sync/oci-image-verification/README.md

```diff
@@ -0,0 +1,156 @@
+# Sample OCI Image Signature Verification with Config Sync
+
+This sample demonstrates how to verify the signature of an OCI image that Config
+Sync is managing. The webhook monitors the `configsync.gke.io/image-to-sync`
```

Comment:
This sample demonstrates how to verify the signature of an OCI image that Config
Sync is managing using [Cosign](https://github.com/sigstore/cosign).

Reply:
Done